openid-data-classes/src/commonMain/kotlin/at/asitplus/openid/Json.kt

```diff
@@ -1,66 +1,13 @@
 package at.asitplus.openid
 
-import at.asitplus.dif.ddcJsonSerializer
-import kotlinx.serialization.SerializationStrategy
 import kotlinx.serialization.json.Json
-import kotlinx.serialization.modules.SerializersModule
-import kotlinx.serialization.modules.overwriteWith
-import kotlinx.serialization.modules.polymorphic
-
-@Suppress("UNCHECKED_CAST")
-private val requestParametersModule = SerializersModule {
-    polymorphic(RequestParameters::class) {
-        subclass(AuthenticationRequestParameters::class, AuthenticationRequestParameters.serializer())
-    }
-    polymorphicDefaultSerializer(
-        RequestParameters::class,
-        defaultSerializerProvider = {
-            when (it) {
-                is AuthenticationRequestParameters -> AuthenticationRequestParameters.serializer() as SerializationStrategy<RequestParameters>
-                else -> throw Exception("Serializer for ${it::class} unknown")
-            }
-        },
-    )
-    polymorphicDefaultDeserializer(
-        RequestParameters::class,
-        defaultDeserializerProvider = { AuthenticationRequestParameters.serializer() },
-    )
-}
-
-@Suppress("UNCHECKED_CAST")
-private val authorizationDetailsModule = SerializersModule {
-    polymorphic(AuthorizationDetails::class) {
-        subclass(OpenIdAuthorizationDetails::class, OpenIdAuthorizationDetails.serializer())
-    }
-    polymorphicDefaultSerializer(
-        AuthorizationDetails::class,
-        defaultSerializerProvider = {
-            when (it) {
-                is OpenIdAuthorizationDetails -> OpenIdAuthorizationDetails.serializer() as SerializationStrategy<AuthorizationDetails>
-                else -> throw Exception("Serializer for ${it::class} unknown")
-            }
-        },
-    )
-    polymorphicDefaultDeserializer(
-        AuthorizationDetails::class,
-        defaultDeserializerProvider = { OpenIdAuthorizationDetails.serializer() },
-    )
-}
-
-/**
- * This serialization module allows de-/serialization of open interfaces defined in this module
- */
-private val baseOpenIdSerializerModule = SerializersModule {
-    include(requestParametersModule)
-    include(authorizationDetailsModule)
-}
 
+// TODO: Review if removable
```

Comment:
i'm in favor of replacing usages of `odcJsonSerializer` with `joseCompliantSerializer`